app/services/catalog/create_approval_request.rb

```diff
@@ -46,12 +46,11 @@ def request_body_from(order_item)
     end
 
     def workflows(portfolio_item)
-      portfolio_item.resolved_workflow_refs
+      []
     end
 
     def create_approval_request(req, order_item)
       ApprovalRequest.create!(
-        :workflow_ref         => req.workflow_id,
```

Comment:
@syncrou @bzwei 
Is this still needed because it is coming back in the response https://github.com/mkanoor/approval-client/blob/master/docs/Request.md?

Reply:
`workflow_id` is a read-only field from the output. It does not exist in the input.

Reply:
@syncrou Makes sense to remove it since a single request could spawn of multiple workflows. So this response coming back from the Approval stores once workflow_id which is in correct since there could be multiples
👍